src/community/gwc-sqlite/src/main/java/org/geoserver/gwc/web/blob/SqliteMultipartFilter.java

```diff
@@ -0,0 +1,22 @@
+package org.geoserver.gwc.web.blob;
+
+import org.geoserver.filters.GeoServerFilter;
+import org.springframework.web.multipart.support.MultipartFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class SqliteMultipartFilter extends MultipartFilter implements GeoServerFilter {
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
+        String path = ((HttpServletRequest) request).getRequestURI();
+        if (path.contains("/sqlite/")) {
```

Comment:
Honestly I cannot remember the exact reason why I make this so permissive but I know he has something to do with the GeoServer integration: https://github.com/nmco/geoserver/blob/c435cf87926c6b2bae9349b19702b752ff1e98f4/src/gwc/src/main/resources/geowebcache-rest-context.xml